WorldProcessor/WorldProcessor.Application/Handlers/WorldProcessing/Mapping/WorldIterationAutoMapperProfile.cs

```diff
@@ -0,0 +1,169 @@
+            IEnumerable<int> source,
+            IPosition destination,
+            ResolutionContext context)
+        {
+            switch (source.Count())
+            {
+                case 2:
+                    {
+                        return new Position2(
+                            source.ElementAt(0),
+                            source.ElementAt(1)
+                        );
+                    }
+                case 3:
+                    {
+                        return new Position3(
+                            source.ElementAt(0),
+                            source.ElementAt(1),
+                            source.ElementAt(2)
+                        );
+                    }
+                default:
+                    {
+                        throw new Exception(
+                            "Undefined type of world dimensions - " +
+                            $"({string.Join(", ", source)})");
+                    }
+            }
+        }
+    }
+
+    public class Position2IEnumerableConverter
+        : ITypeConverter<IPosition, IEnumerable<int>>
+    {
+        public IEnumerable<int> Convert(
+            IPosition source,
+            IEnumerable<int> destination,
+            ResolutionContext context)
+            => source.GetCoordinatesList();
+    }
+
+    public class WorldConfigDto2WorldConfigConverter
+        : ITypeConverter<WorldConfigDto, WorldConfig>
+    {
+        public WorldConfig Convert(WorldConfigDto source, WorldConfig destination, ResolutionContext context)
+        {
+            return new WorldConfig()
+            {
+                Id = source.Id,
+                TileShape = source.TileShape,
+                Tiles = source.Tiles
+                    .Select(dto => context.Mapper.Map<TileConfigDto, TileConfig>(dto))
+                    .ToList()
+            };
+        }
+    }
+
+    public class WorldDto2WorldConverter
+        : ITypeConverter<WorldDto, World>
+    {
+        public World Convert(
+            WorldDto source,
+            World destination,
+            ResolutionContext context)
+        {
+            var result = new World()
+            {
+                ConfigId = source.ConfigId,
+                Seed = source.Seed,
+                Epoch = source.Epoch,
+                TileShape = source.TileShape
+            };
+
+            result.Dimensions = context.Mapper
+                .Map<IEnumerable<int>, IPosition>(source.Dimensions);
+
+            result.Tiles = new Dictionary<IPosition, Tile>();
+
+            foreach(var kvp in source.Tiles)
+            {
+                IPosition tilePosition = context.Mapper
+                    .Map<IEnumerable<int>, IPosition>(
+                        kvp.Key
+                            .Split(",")
+                            .Select(str => int.Parse(str)));
+
+                if(tilePosition.GetType() != result.Dimensions.GetType())
+                {
+                    throw new Exception(
```

Comment:
Yes, it will continue to work. Because this application is stateless, the exception will not cause problems. But this is a good point. I will add exception handlers. While they will be simple and will be improved in the near future.